src/helpers/regexps.ts

```diff
@@ -0,0 +1,4 @@
+export const emailRegexp: RegExp = /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
```

Comment:
Wow!
https://david-gilbertson.medium.com/the-100-correct-way-to-validate-email-addresses-7c4818f24643
:)